real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/localizers/HighTrustGPSLocalizer.java

```diff
@@ -42,6 +43,7 @@ public HighTrustGPSLocalizer(){
     	buggyFrameGpsY = 0.0;
```

Comment:
Note that subscriber callbacks all run on different threads. you may hit races if you don't plan for this,